packages/cf-builder-pagination/src/PaginationBuilder.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Pagination, PaginationItem } from 'cf-component-pagination';
-import Icon from 'cf-component-icon';
+import Icon from '../../cf-component-icon/src';
```

Comment:
I think you just forgot to run `npm run build` before `npm test`. Stuff is already symlinked by lerna but it's pointing to `packages/cf-component-icon/lib/index.js`, so you need to rebuild before running tests.

> This is probably wrong since this will break these components.

Yep.